python/plugins/db_manager/db_plugins/postgis/info_model.py

```diff
@@ -84,7 +85,7 @@ def generalInfo(self):
             if table_priv[0]:
                 privileges.append("select")
 
-                if self.table.rowCount is not None or self.table.rowCount >= 0:
+                if self.table or self.table.rowCount is not None or self.table.rowCount >= 0:
```

Comment:
@slarosa does this line make sense or should this be an `and` rather than an `or`? If the table is defined, the other checks will not be executed. But the other checks only make sense if the table is defined.

Reply:
true, ```and``` is the operator to use here, thanks.